src/functionalTest/resources/scenarios/RIA-3313-appellant-ftpa-submitted-ho-notification-failure.json

```diff
@@ -0,0 +1,100 @@
+{
+  "description": "RIA-3313 appellant ftpa application submitted Home Office notification failure",
+  "request": {
+    "uri": "/asylum/ccdAboutToSubmit",
+    "credentials": "LegalRepresentative",
+    "input": {
+      "id": 3313,
+      "eventId": "applyForFTPAAppellant",
+      "state": "ftpaSubmitted",
+      "caseData": {
+        "template": "minimal-appeal-submitted.json",
+        "replacements": {
+          "ariaListingReference": "LP/12345/2019",
+          "listCaseHearingCentre": "bradford",
+          "homeOfficeFtpaAppellantInstructStatus": "FAIL"
```

Comment:
Do these test need Launch Darkly check ?

Reply:
The changes are independent and can be run with or without the flag.